x/subkeys/internal/types/tx/codec.go

```diff
@@ -0,0 +1,8 @@
+package tx
```

Comment:
Yes, I was uncomfortable pulling in auth/types (and auth/keeper and auth/ante) into the `x/subkeys` package root. I highly doubt this will be an import cycle (auth should never import this code), but it did seem to violate much of the reasoning of the current design and separation of concerns.

I could not implement this code without the "internal" import or copying all the code from ante. If I get your blessing, I can move this into types (from types/tx), and also expose aliases to `internal/ante` on the top-level module.

Reply:
I think the later is fine 👍